.github/workflows/Build and test.yml

```
@@ -33,36 +33,28 @@ jobs:
-        mkdir $PWD/avr-toolchain &&
-        tar -xf avr-gcc-13.2.0-x64-linux.tar.gz -C $PWD/avr-toolchain
+    - name: Build docker container
+      run: >
+        docker build -t avr-gcc-toolchain:latest -f Dockerfile . --platform=linux/amd64
 
     - name: Configure CMake
-      run: 
-        export PATH="$PWD/avr-toolchain/avr-gcc-13.2.0-x64-linux/bin:$PATH" &&
-        cmake -B ${{github.workspace}}/build -DCMAKE_BUILD_TYPE=${{matrix.build_type}} -S . -DNEXTAG_EMBEDDED_PLATFORM_CONFIGURE_AS_UNO=ON
+      run: >
+        docker run --rm 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)